src/Bicep.Core/Syntax/ImportSpecification.cs

```diff
@@ -41,52 +41,54 @@ public class ImportSpecification : ISymbolNameSource
             RegexOptions.ECMAScript | RegexOptions.Compiled);
 
         private static readonly Regex RepositoryNamePattern = new(
-            @"^\S*[:\/](?<name>az)$",
+            @"^\S*[:\/](?<name>\S+)$",
```

Comment:
It might be a good chance to convert these to source generated Regex insntaces with `GeneratedRegexAttribute`.